test/integration/apiserver/admissionplugins/shootvalidator/shootvalidator_suite_test.go

```diff
@@ -314,6 +314,16 @@ var _ = BeforeSuite(func() {
 						Namespace: "some-namespace",
 					},
 				},
+				Internal: &gardencorev1beta1.SeedDNSProviderConfig{
+					Type:   "local",
```

Comment:
Cosmetic suggestion:

```suggestion
					Type:   "provider",
```

so that it matches the provider type of the dns provider.